bob-common/src/bob_client.rs

```diff
@@ -40,9 +41,18 @@ pub mod b_client {
             node: Node,
             operation_timeout: Duration,
             metrics: BobClientMetrics,
+            ca_cert_path: Option<&str>,
             local_node_name: String,
         ) -> Result<Self, String> {
-            let endpoint = Endpoint::from(node.get_uri()).tcp_nodelay(true);
+            let mut endpoint = Endpoint::from(node.get_uri());
+            if let Some(ca_cert_path) = ca_cert_path {
+                let cert_bin = fs::read(ca_cert_path).expect("can not read ca certificate from file");
+                let cert = Certificate::from_pem(cert_bin);
+                let tls_config = ClientTlsConfig::new().domain_name("bob").ca_certificate(cert);
```

Comment:
As I understand, the domain should be configurable, because certificate depends on the domain name